HHS/Head-Start-TTADP
# [TTAHUB-2721] Change unlock report permissions

## Description of change

Per user request - if a user is an approver and the creator, and the report is in "needs action", allow that user to reset the report to draft.

## How to test

- Load prod data
- Impersonate user 160
- open report 40979
- reset report to draft

Or, with your own database and environment, set yourself as an approver and creators on a report and then reset that report to draft. Confirm that the functionality doesn't exist for approvers who aren't creators, or creators who aren't approvers (on a given report)

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2721


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
